addons/web/static/src/js/widgets/data_export.js

```diff
@@ -56,10 +56,19 @@ var DataExport = Dialog.extend({
                 }
             }
         },
-        'dblclick .o_export_tree_item': function(e) {
+        'dblclick .o_export_tree_item:not(.haschild)': function(e) {
```

Comment:
Why not just remove the o_export_tree_item from rows with children?

Reply:
We can't remove `.o_export_tree_item` class because it's already used by its children at many place.